client/layout/guided-tours/config.js

```diff
@@ -118,10 +120,122 @@ const tours = {
+			next: isMobile() ? 'mobileCloseSearch' : 'filter',
+		},
+		mobileCloseSearch: {
+			text: 'Close the search',
+			type: 'ActionStep',
+			target: '.themes__search-card .search-close__icon',
+			placement: 'below',
+			arrow: 'top-left',
+			next: 'mobileFilter',
+		},
+		filter: {
+			text: 'Here you can filter between free and premium themes. Try filtering by _free_ themes now.',
+			type: 'ActionStep',
+			target: 'themes-tier-dropdown',
+			placement: 'above',
+			continueIf: state => {
+				const params = getQueryParams( state );
+				return params && params.tier === 'free';
+			},
+			arrow: 'bottom-right',
+			next: 'choose-theme',
+		},
+		mobileFilter: {
+			text: 'Here you can filter between free and premium themes. Try filtering by _free_ themes now.',
+			type: 'ActionStep',
+			target: '.themes__search-card .section-nav__mobile-header',
+			placement: 'above',
+			showInContext: () => isMobile(),
+			continueIf: state => {
+				const params = getQueryParams( state );
+				return params && params.tier === 'free'; // probably want to allow any selection, and check it's changed
+			},
+			arrow: 'top-right',
+			next: 'choose-theme',
+		},
+		'choose-theme': {
+			text: "Tap on a theme to see more details — such as screenshots, the theme's features, or a preview.",
+			type: 'ActionStep',
+			placement: 'center',
+			showInContext: state => getSectionName( state ) === 'themes',
+			continueIf: state => getSectionName( state ) === 'theme',
+			next: 'tab-bar',
+		},
+		'tab-bar': {
+			text: 'Here you can take a look at more screenshots of the theme, read about its features, or get help on how to use it.',
+			type: 'BasicStep',
+			placement: 'center',
+			target: '.section-nav',
+			showInContext: state => getSectionName( state ) === 'theme',
+			next: 'live-preview',
+		},
+		'live-preview': {
+			text: 'Tap here to see a _live demo_ of the theme.',
+			type: 'ActionStep',
+			placement: 'below',
+			target: 'theme-sheet-preview',
+			showInContext: state => getSectionName( state ) === 'theme',
+			arrow: 'top-left',
+			next: 'close-preview',
+		},
+		'close-preview': {
+			target: '.web-preview.is-visible [data-tip-target="web-preview__close"]',
+			arrow: 'left-top',
+			type: 'ActionStep',
+			placement: 'beside',
+			icon: 'cross-small',
+			text: "This is the theme's preview. Take a look around! Then tap to close the preview.",
+			showInContext: state => state && isPreviewShowing( state ),
+			continueIf: state => ! isPreviewShowing( state ),
+			next: 'back-to-list',
+		},
+		'back-to-list': {
+			arrow: 'left-top',
+			type: 'ActionStep',
+			target: '.theme__sheet-action-bar .header-cake__back.button',
+			placement: 'beside',
+			icon: 'arrow-left',
+			text: 'You can go back to the themes list here.',
+			continueIf: state => getSectionName( state ) === 'themes',
+			next: 'finish',
+		},
+		finish: {
+			placement: 'center',
+			text: "That's it!",
+			showInContext: state => getSectionName( state ) === 'themes',
```

Comment:
Again, a minor suggestion, which we can address later, but there are two functions we define repeatedly, which we could define in the module scope: `isSectionThemes`, `isSectionTheme`.